content/post/know-your-aws-community-day-india-2020-speaker-yogesh-sharma.md

```diff
@@ -0,0 +1,27 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T19:02:16Z
+description = "Interested to know more about Yogesh Sharma? Read on."
+feature_image = "/images/yogesh.PNG"
+tags = ["AWS", "AWSCommunityDay2020", "Speaker", "Kinesis", "Fargate", "CFN"]
```

Comment:
Speaker tag should be first

Reply:
It's not able to come first. I have tried many times.

I also tried by deleting all tags and starting with speaker tag but still, it's not taking